src/subdomains/staking/domain/entities/staking-balance.entity.ts

```diff
@@ -9,7 +9,7 @@ import { StakingBalances } from '../../application/services/staking.service';
 @Entity()
 @Index((sb: StakingBalance) => [sb.staking, sb.asset], { unique: true })
 export class StakingBalance extends IEntity {
-  @ManyToOne(() => Staking, (staking) => staking.balances, { nullable: false })
+  @ManyToOne(() => Staking, (staking) => staking.balances, { nullable: false, eager: true })
```

Comment:
Not a good idea, balances are already loaded eagerly on staking.

Reply:
_Eager relations can only be used on one side of the relationship, using eager: true on both sides of relationship is disallowed._
https://orkhan.gitbook.io/typeorm/docs/eager-and-lazy-relations